templates/new/mix.exs

```diff
@@ -73,9 +73,8 @@ defmodule <%= app_module %>.MixProject do
   defp aliases("host"), do: []
 
   defp aliases(_target) do
-    [
-      "deps.precompile": ["nerves.precompile", "deps.precompile"],
-      "deps.loadpaths": ["deps.loadpaths", "nerves.loadpaths"]
-    ]
+    Nerves.Bootstrap.merge_aliases([
+      # You can add your own custom mix aliases here
+    ])
```

Comment:
What do you think of this?

Reply:
I'm good with the general idea. I probably would write something like this.
```
  [
    # Add custom mix aliases here
  ] 
  |>  Nerves.Bootstrap.add_aliases()
```
I like the word "add" over "merge" for what the function will do. Functions with "merge" in their name make me expect to pass two arguments (i.e., the things to be merged.) I could also be fine with "insert_aliases" or "inject_aliases".